docs/usage.adoc

```diff
@@ -10,17 +10,17 @@ Let's see some examples:
 
 *You want to run only tests that you've just added or modified locally*
 
-`$ mvn clean test -Dsmart.testing.mode=selecting -Dsmart.testing="new, changed"`
+`$ mvn clean test -Dconst:core/src/main/java/org/arquillian/smart/testing/Configuration.java[name="SMART_TESTING_MODE"]=selecting -Dconst:core/src/main/java/org/arquillian/smart/testing/Configuration.java[name="SMART_TESTING"]="new, changed"`
 
 *You want to run all tests but given priority to the latest tests added or modified*
 
-`$ mvn clean test -Dsmart.testing.mode=ordering -Dscm.last.changes=1 -Dsmart.testing="new, changed"`
+`$ mvn clean test -Dconst:core/src/main/java/org/arquillian/smart/testing/Configuration.java[name="SMART_TESTING_MODE"]=ordering -Dconst:core/src/main/java/org/arquillian/smart/testing/scm/ScmRunnerProperties.java[name="LAST_COMMITS"]=1 -Dconst:core/src/main/java/org/arquillian/smart/testing/Configuration.java[name="SMART_TESTING"]="new, changed"`
 
-TIP: `smart.testing.mode` is not mandatory in case of `ordering` because it is the **default** mode.
+TIP: `const:core/src/main/java/org/arquillian/smart/testing/Configuration.java[name="SMART_TESTING_MODE"]` is not mandatory because `const:core/src/main/java/org/arquillian/smart/testing/Configuration.java[name="DEFAULT_MODE"]` is the **default** mode.
```

Comment:
You have changed the wrong/old line:

> smart.testing.mode is not mandatory in case of ordering because it is the default mode.

to:

> smart.testing.mode is not mandatory because selecting is the default mode.

but in the description of this command is:

> You want to run all tests but given priority to the latest tests added or modified

In other words, the user wants to run *all* tests, so the mode is mandatory and has to contain `ordering`

Reply:
Don't get what you mean

Reply:
In the description of the command is:

You want to run **all** tests....

which means that you should specify the ordering mode inside of the command. 
This part was probably written at that time when we had `ordering` mode as the default one. When we changed it to `selecting` we probably forgot to change also the documentation...

Reply:
and yeah, the preceding example:

> You want to run only tests that you’ve just added or modified locally
`$ mvn clean test -Dsmart.testing.mode=selecting -Dsmart.testing.report.dir="new, changed"`

 could be changed as well. The mode is not necessary here as selecting is the default one :-)

Reply:
And the next one as well: 

> You want to run only tests that validates new or modified business classes locally
`$ mvn clean test -Dsmart.testing.mode=selecting -Dsmart.testing.report.dir="affected"`